src/__tests__/forms/FilterProvincie.cy.ts

```diff
@@ -1,18 +1,28 @@
-import FilterProvincie from '../components/smart/FilterProvincie.vue';
 import { defineComponent, ref } from 'vue';
+import { OeFilterProvincie } from '@components/forms';
 import type { IProvincie } from '@models/locatie';
 
 describe('FilterProvincie', () => {
+  beforeEach(() => {
+    cy.on('uncaught:exception', (err) => {
```

Comment:
Kan dit niet in ne cypress config gezet worden dat hij die check moet doen voor elke component?